src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,18 +1,12 @@
 import * as React from 'react';
-import { ICheckboxProps } from './Checkbox.Props';
+import { ICheckboxProps, ICheckbox } from './Checkbox.Props';
```

Comment:
There is a lint rule for this i have disabled. I am mixed on if this is a nuisance or useful.